app/models/models.js

```diff
@@ -2196,20 +2191,32 @@ YUI.add('juju-models', function(Y) {
      * @return {Object} export object suitable for serialization.
      */
     exportDeployer: function() {
-      var serviceList = this.services,
-          relationList = this.relations,
-          defaultSeries = this.environment.get('defaultSeries'),
-          result = {
-            envExport: {
-              services: {},
-              relations: []
-            }
-          };
+      var defaultSeries = this.environment.get('defaultSeries'),
```

Comment:
It's currently precise. Changing this will not be a drive-by as it'll break a number of tests and change how the charmstore search results work. while in sandbox mode.